package.json

```diff
@@ -73,9 +73,10 @@
     "lavamoat:build": "lavamoat development/build/index.js --policy lavamoat/build-system/policy.json --policyOverride lavamoat/build-system/policy-override.json",
     "lavamoat:build:auto": "yarn lavamoat:build --writeAutoPolicy",
     "lavamoat:debug:build": "yarn lavamoat:build --writeAutoPolicyDebug --policydebug lavamoat/build-system/policy-debug.json",
-    "lavamoat:background:auto": "./development/generate-lavamoat-policies.sh",
-    "lavamoat:background:auto:dev": "./development/generate-lavamoat-policies.sh --dev",
```

Comment:
Maybe removing this script can be broken out in a separate change that follows the rest of this?

Reply:
I don't think that's worth the effort, although I did update the PR description to call out this removal.